docs/plugins/rate-limit/how-to.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: How to Configure the Rate Limit Plugin
+sidebar_position: 2
+description: "Learn how to configure the Rate Limit Plugin for Hasura DDN."
+keywords:
+  - hasura plugins
+  - rate limit plugin
+  - plugins architecture
+  - engine plugins
+  - how-to
+  - guide
+---
+
+# How to Configure the Rate Limit Plugin
+
+## Introduction
+
+The [rate limit plugin](https://github.com/hasura/engine-plugin-rate-limit) adds the ability to limit the number of
+requests that can be made to DDN in a given time period.
+
+## Setting up for local development
+
+We can set up everything we need for local development through Docker. Add the following entries to your `compose.yaml`
+file:
```

Comment:
As a user, I'm confused as to which compose file I should be adding this to. Is it the one at the root of my supergraph? Are these just new items in the `services` array?

Let's hold the user's hand a little more and be very prescriptive so there's no chance for them to get derailed or confused early on.